<!-- 
         Velocity smoother
-->
<launch>
  <node pkg="nodelet" type="nodelet" name="velocity_smoother" args="load yocs_velocity_smoother/VelocitySmootherNodelet /mobile_base_nodelet_manager">
    <rosparam file="$(find turtlebot_bringup)/param/defaults/smoother.yaml" command="load"/>
    <remap from="velocity_smoother/smooth_cmd_vel" to="/cmd_vel_mux/input/navi"/>

    <!-- Robot velocity feedbacks; use the default base configuration -->
    <remap from="velocity_smoother/odometry" to="/odom"/>
    <remap from="velocity_smoother/robot_cmd_vel" to="/mobile_base/commands/velocity"/>

    <!-- connecting between reinforcement/cmd_vel in mobile_base_nodelet and smoother/raw_cmd_vel in camera_nodelet -->
<!--     <remap from="velocity_smoother/raw_cmd_vel" to="/mobile_base_nodelet_camera_nodelet/raw_cmd_vel"/> -->

    <!-- Configure velocity smoother to allow more aggressive motion -->
    <param name="accel_lim_v"  value="0.6"/>
    <param name="accel_lim_w"  value="5.5"/>
    <param name="decel_factor" value="2.0"/>
  </node>
</launch>
